src/components/sbb-timetable-row/sbb-timetable-row.helper.tsx

```diff
@@ -96,3 +106,94 @@ export const renderStringProduct = (vehicleName: string, line: string): JSX.Elem
     </span>
   );
 };
+
+const isReachable = (legs: PTRideLeg[]): boolean => {
+  return legs?.some((leg) => leg.serviceJourney?.serviceAlteration?.reachable === true);
+};
+
+const isRedirected = (legs: PTRideLeg[]): boolean => {
+  return legs?.some((leg) => leg.serviceJourney?.serviceAlteration?.redirected === true);
+};
+
+const isUnplannedStop = (legs: PTRideLeg[]): boolean => {
+  return legs?.some(
+    (leg) => !!leg.serviceJourney?.serviceAlteration?.unplannedStopPointsText === true
+  );
+};
+
+export const sortSituation = (situations: PtSituation[]): PtSituation[] => {
+  const priorities = {
+    DISTURBANCE: 0,
+    INFORMATION: 1,
+    DELAY: 2,
+    TRAIN_REPLACEMENT_BY_BUS: 3,
+    CONSTRUCTION_SITE: 4,
+    END_MESSAGE: 5,
+  };
+
+  return situations
+    ?.slice()
```

Comment:
return [...situations].sort(...) ?

Reply:
no conditional chaining required (?), because situations is not an optional argument